test/EndToEndSpec.hs

```diff
@@ -1436,6 +1436,33 @@ spec = around_ (inTempDirectoryNamed "foo") $ do
             ghc-options: -main-is Foo
           |]
 
+      describe "rts-options" $ do
+        it "maps rts-options to ghc-options" $ do
+          [i|
+          executable:
+            main: Main.hs
+            rts-options: -s -N
+          |] `shouldRenderTo` executable_ "foo" [i|
+            main-is: Main.hs
+            ghc-options: "-with-rtsopts -s -N"
```

Comment:
Do these need to be quoted, like `-with-rtsopts='-s -N'`? Or perhaps the flag can be duplicated, like `-with-rtsopts=-s -with-rtsopts=-N`.

Reply:
From reading the motivation for this PR, this it's what I capture: Giving `--with-rtsopts` multiple times is not accumulative. That's the motivation for this PR in the first place.

Regarding quoting, I assume that what this PR currently does works with Cabal. I haven't  tried it again just yet, but pretty sure that I tried when I originally wrote this.


`--with-rtsopts='-s -N'` may work as well, somebody would need to try. I'm fine with any thing that works here, I'm not very opinionated on this.

Reply:
This Cabal issue is very related: https://github.com/haskell/cabal/issues/4818

Reply:
Thanks @tfausak for doing all the heavy lifting.

I rarely talk about it in public, but I have had issues with RSI for several years now. when it flares up, I have to stay away from the computer. This is the reason why sometimes I'm less responsive than what I would hope to be.

The TL;DR of it is that I am away from the computer. We still need the  read me update, change  log entry and version bump. I can try to take care of it eventually. But if somebody wants to lend a hand, then that's still very much appreciated.

Releases are fully automated once the version bump hits main.

Reply:
I'd be happy to do the _README.md_ update etc today.